### ISSUE TITLE
Tray icon

### ISSUE THREAD

**Issue by tbodt** [d4nj1/TLPUI#156]
Sometimes you want to manually override TLP to run in battery or ac mode, and this is easily done from the terminal, but it would be less scary for TLPUI to offer a status item for this.

**Comment by tbodt** [comment]
Planning to write a PR for this

**Comment by d4nj1** [comment]
Good idea. What front end are you using? KDE? Gnome? XFCE? Something else?

**Comment by d4nj1** [comment]
Closed for now due to lack of feedback.
